ocs_ci/ocs/platform_nodes.py

```diff
@@ -1685,6 +1686,67 @@ def read_default_config(self, default_config_path):
         return default_config_dict
 
 
+class IBMPowerNodes(BaremetalNodes):
```

Comment:
I'm wondering about the naming and inheritance here. I can't see anything specific to Power, and it seems to me this could be useful for anyone doing an libvirt deployment. Perhaps it ought to be `LibvirtNodes`?

Reply:
I put together these changes with the future in mind, for Power currently do things on libvirt, but we certainly would like to do these in PowerVS (IBM Cloud) and Power Baremetal as well one day. When we do that, we may reuse part of Baremetal code that already exists.

Reply:
I put together these changes with the future in mind, for Power currently does things on libvirt, but we certainly would like to do these in PowerVS (IBM Cloud) and Power Baremetal as well one day. When we do that, we may reuse part of Baremetal code that already exists.